src/main/java/bookstore/dto/cart/CartItemCreateDto.java

```diff
@@ -0,0 +1,12 @@
+package bookstore.dto.cart;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Positive;
+
+public record CartItemCreateDto(
+        @Positive
+        Long book_id,
```

Comment:
```suggestion
        Long bookId,
```

Check other variables